hack/create-user-workspace.sh

```diff
@@ -16,6 +16,7 @@ SERVICE_NAME=${1}
 ROOT_WORKSPACE=${ROOT_WORKSPACE:-"root"}
 APPSTUDIO_WORKSPACE=${APPSTUDIO_WORKSPACE:-"redhat-appstudio"}
 HACBS_WORKSPACE=${HACBS_WORKSPACE:-"redhat-hacbs"}
+PIPELINE_SERVICE_WORKSPACE=${PIPELINE_SERVICE_WORKSPACE:-"pipeline-service"}
```

Comment:
this is not the right workspace name for kcp-stable:
```suggestion
PIPELINE_SERVICE_WORKSPACE=${PIPELINE_SERVICE_WORKSPACE:-"redhat-pipeline-service-compute"}
```